File: hcgcalidad_services-master/hcgcalidad_services-master/HCGCALIDADSERVICES/Models/Firma.cs
using System;
using System.Collections.Generic;

namespace HCGCALIDADSERVICES.Models
{
    public partial class Firma
    {
        public Firma()
        {
            DetalleFirma = new HashSet<DetalleFirma>();
        }

        public int FirmaId { get; set; }
        public string FirmaNombre { get; set; }
        public string FirmaCargo { get; set; }
        public string FirmaCodigo { get; set; }
        public string FirmaCorreo { get; set; }
        public int? Repetido { get; set; }

        public ICollection<DetalleFirma> DetalleFirma { get; set; }
    }
}

File: hcgcalidadapp/lib/src/modelos/firma.dart
// To parse this JSON data, do
//
//     final firma = firmaFromJson(jsonString);

import 'dart:convert';

Firma firmaFromJson(String str) => Firma.fromJson(json.decode(str));

String firmaToJson(Firma data) => json.encode(data.toJson());

class Firma {
  Firma({
    this.firmaId,
    this.firmaNombre,
    this.firmaCargo,
    this.firmaCorreo,
    this.firmaCodigo
  });

  int firmaId;
  String firmaCodigo;
  String firmaNombre;
  String firmaCargo;
  String firmaCorreo;

  factory Firma.fromJson(Map<String, dynamic> json) => Firma(
    firmaId: json["firmaId"],
    firmaCodigo: json["firmaCodigo"],
    firmaNombre: json["firmaNombre"],
    firmaCargo: json["firmaCargo"],
    firmaCorreo: json["firmaCorreo"]
  );

  Map<String, dynamic> toJson() => {
    "firmaId": firmaId,
    "firmaCodigo" : firmaCodigo,
    "firmaNombre": firmaNombre,
    "firmaCargo": firmaCargo,
    "firmaCorreo": firmaCorreo
  };
}
